### PR TITLE
xmlsec1: new package at 1.2.29

### PR DESCRIPTION
**Pull request by artemkobets**
## Description of package
XML security library. Dependency for `oath-toolkit`
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/xmlsec1/sources [hunk 1]
```diff
@@ -0,0 +1 @@
+http://www.aleksey.com/xmlsec/download/xmlsec1-1.2.29.tar.gz
```
Comment (dylanaraps) [inline community/xmlsec1/sources]: No `https`?

File: community/xmlsec1/depends [hunk 2]
```diff
@@ -0,0 +1,2 @@
+libressl
```
Comment (artemkobets) [inline community/xmlsec1/depends]: I'm not sure about dependencies here.
According to FAQ https://www.aleksey.com/xmlsec/faq.html xmlsec can be built against
different crypto libraries (openssl/libressl, gnutls, libgcrypt, nss).
I only added `libressl` to depends file to satisfy minimum requirements, but would it make more sense to include `gnutls`, `libgcrypt` and `nss` here as well?

File: community/xmlsec1/sources (hunk 1 above)
Comment (artemkobets) [reply community/xmlsec1/sources]: oops, fixed

File: community/xmlsec1/depends (hunk 2 above)
Comment (dylanaraps) [reply community/xmlsec1/depends]: It would not. You should be able to use configure flags to limit it to libressl, if not it is fine as-is. :+1:

Comment (artemkobets) [reply community/xmlsec1/depends]: Added flags and a missing `libtool` dependency. Should be good now

**Comment by dylanaraps** [comment]
:+1:

**Pull request merged**